orchestrator/utils/redis.py

```diff
@@ -86,8 +86,8 @@ def delete_subscription_from_redis(
 ) -> Callable[[Callable[..., Any]], Callable[..., Any]]:
     def _delete_subscription_from_redis(func: Callable[..., Any]) -> Callable[..., Any]:
         @functools.wraps(func)
-        def wrapper(*args: tuple, **kwargs: dict[str, Any]) -> Any:
-            data = func(*args, **kwargs)
+        async def wrapper(*args: tuple, **kwargs: dict[str, Any]) -> Any:
+            data = await func(*args, **kwargs)
```

Comment:
Why is this changed to async?  In surf orchestrator this function is used as a decorator on sync functions. And even if we change those to async, then this will break it for other users